src/app/app.module.ts

```diff
@@ -49,6 +50,7 @@ export const translateHttpLoaderFactory = (httpClient: HttpClient) =>
       },
     }),
     ReactiveFormsModule,
+    NgOtpInputModule,
```

Comment:
remove NgOtpInputModule and its import here you have add in otp module also so not needed here